<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>terry_calculator</class>
 <widget class="QMainWindow" name="terry_calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>terry_calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QPushButton" name="Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Multiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="MemAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Add">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="MemClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Divide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Subtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: gray;
	border: 1px solid gray;
	padding: #ffffff;
}	
</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="MemGet">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton.buttonGroup_2 {
	background-color: #66cd00;
	border: 1px solid gray;
	
	padding: 5px;
	border-radius: 30px;
}	

QPushButton.buttonGroup_2:hover {
	background-color: #737373;
	
	padding: 5px;
	border-radius: 30%;
}


QPushButton.buttonGroup_2:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="ChangeNumberSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton.buttonGroup {
	background-color: #4c4c4c;

	border: 1px solid gray;
	border-radius: 30%;
	padding: 5px;
}	

QPushButton.buttonGroup:hover {
	background-color: #737373;
	background-radius: 30%;
	padding: 5px;
}

QPushButton.buttonGroup:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	border-radius: 30px;
	padding: 5px;
}	


</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
   <zorder>Button4</zorder>
   <zorder>Button6</zorder>
   <zorder>Button5</zorder>
   <zorder>Divide</zorder>
   <zorder>Button7</zorder>
   <zorder>MemClear</zorder>
   <zorder>Button8</zorder>
   <zorder>Multiply</zorder>
   <zorder>Button9</zorder>
   <zorder>MemAdd</zorder>
   <zorder>Button1</zorder>
   <zorder>Clear</zorder>
   <zorder>Button0</zorder>
   <zorder>Button2</zorder>
   <zorder>Button3</zorder>
   <zorder>ChangeNumberSign</zorder>
   <zorder>Subtract</zorder>
   <zorder>MemGet</zorder>
   <zorder>Equals</zorder>
   <zorder>Display</zorder>
   <zorder>Add</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
